lte/gateway/c/core/oai/test/spgw_task/spgw_test_util.cpp

```diff
@@ -62,6 +62,29 @@ bool is_num_sessions_valid(
   return true;
 }
 
+bool is_num_ue_contexts_valid(uint64_t imsi64, int expected_num_ue_contexts) {
```

Comment:
No need for Imsi here

```suggestion
bool is_num_ue_contexts_valid(int expected_num_ue_contexts) {
```

Reply:
Removed